GoMeet/src/main/java/com/google/sps/servlets/GoogleCalendarTimesServlet.java

```diff
@@ -0,0 +1,167 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.TimePeriod;
+import com.google.gson.Gson;
+import com.google.sps.data.AutoProposeTimes;
+import com.google.sps.data.ErrorMessages;
+import com.google.sps.data.MeetingEventFields;
+import com.google.sps.data.ServletUtil;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/gcal-find-times")
+public class GoogleCalendarTimesServlet extends HttpServlet {
+  // TODO: Load the API_KEY from a file.
+  private static String API_KEY = "";
+  
+  /** Field names used for accessing pairs in the query string 
+    * sent to this servlet */
+  public static class QueryStringFieldNames {
```

Comment:
optional: Since I believe this class isn't being referred to outside of your tests, you could make this private.

Reply:
It's referred to in the test cases 😅

Reply:
Oh my, oops. I must have missed that. Thanks for correcting me.

Reply:
No worries! It's all good 😊